x-pack/plugins/ml/public/application/services/ml_api_service/data_frame_analytics.ts

```diff
@@ -47,11 +47,12 @@ interface DeleteDataFrameAnalyticsWithIndexResponse {
 }
 
 export const dataFrameAnalytics = {
-  getDataFrameAnalytics(analyticsId?: string) {
+  getDataFrameAnalytics(analyticsId?: string, excludeGenerated?: boolean) {
     const analyticsIdString = analyticsId !== undefined ? `/${analyticsId}` : '';
     return http<GetDataFrameAnalyticsResponse>({
       path: `${basePath()}/data_frame/analytics${analyticsIdString}`,
       method: 'GET',
+      query: { excludeGenerated },
```

Comment:
`excludeGenerated ` shouldn't be added to the query if it is `undefined`

Reply:
Removed here as it's no longer needed 
 https://github.com/elastic/kibana/pull/88898/commits/e9ad8f16c6e349a3d7bbc5199ad6d12de0f1bb72